main/Util/TempFile.cs

```diff
@@ -18,11 +18,15 @@ public class TempFile
          */
         public static FileInfo CreateTempFile(String prefix, String suffix)
         {
-
-            if (dir == null)
+            if (string.IsNullOrWhiteSpace(dir))
             {
-                dir = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), "poifiles")).FullName;
+                string tempDir = Path.Combine(Path.GetTempPath(), "poifiles");
+                dir = Directory.CreateDirectory(tempDir).FullName;
```

Comment:
CreateDirectory always called so shouldn't actually be dir be assigned tempdirs value?

Reply:
I didn't change much code, but `dir` variable doesn't needed at all